### DISCUSSION TITLE
How to export "User groups" setting

### DISCUSSION THREAD

**Discussion by mmierzejewski** [shridhar-tl/jira-assistant#220] [Q&A] [answered]
Hi,
How can I export "User groups" setting from my browser to the other one? Is it possible to do it easy?
I will use it as a configuration setting backup my main browser.

**Comment by shridhar-tl** [comment, marked as answer]
Hi @mmierzejewski - As of now, their are no options to export any settings. But currently I am working on major refactoring to support multiple features like backup & sync across browsers (#67) , BETA testing (#221). So in around a week's time, you can expect this feature to be available.

**Reply by mmierzejewski** [reply]
Hi, Thanks for good news! I'm waiting to use that solution when it is deployed.

**Comment by shridhar-tl** [comment]
This issue has been resolved long back and hence closing this ticket. Please feel free to reopen it if you have any other queries.
